### ISSUE TITLE
Browser Doesn't Open

### ISSUE THREAD

**Issue by punktiktok** [Alex313031/Thorium-Win#253]
**System Details**
 - OS: Windows 11]
 - Thorium Version AVX2

**Problem**
After Opening The  exe it doesn't do anything

**Comment by Aedon** [comment]
Run the exe as an administrator, wait for some time (you can see something's happening in the task manager). Then it should be searchable in windows search.

**Comment by kLiHz** [comment]
Yes the installer needs some time to finish. Also sometimes it seems that I need to run the installer more than once.

By the way, I guess the installer can be run from command line l. If so, I guess it would be easier to tell if it's finished already or not.

**Comment by punktiktok** [comment]
> Run the exe as an administrator, wait for some time (you can see something's happening in the task manager). Then it should be searchable in windows search.

in Task manager it doesn't show anything

**Comment by gz83** [comment]
Please try the latest version 138. If you still have problems, please let us know.
